src/objects/media/media.stories.mdx

```diff
@@ -41,13 +50,23 @@ This pattern is composed of three elements:
 The order of `o-media__object` and `o-media__content` will not affect this pattern's visual appearance. To display the object on the right, use the `o-media--reverse` modifier.
 
 <Canvas>
-  <Story name="Image Reversed">
-    {imageDemo({
-      imgSrc: meganProfileImage,
-      text: text('text', exampleText),
-      imgSize: number('image size', 64, imgSizeOptions),
-      reverse: true,
-    })}
+  <Story
+    name="Image Reversed"
+    argTypes={{
+      imgSrc: {
+        type: { name: 'string', required: true },
+        defaultValue: meganProfileImage,
+      },
+      text: { type: { name: 'string' }, defaultValue: exampleText },
+      imgSize: {
+        type: { name: 'number' },
+        control: imgSizeOptions,
+        defaultValue: 64,
+      },
+      reverse: { type: { name: 'boolean' }, defaultValue: true },
+    }}
```

Comment:
Is there a way using Controls that we can cut down on this repetition and just show a different version of the first story with `reverse` set to `true`?

Or alternatively, could we expose the list of controls in the Docs view so we don't need to manually show that example, but have a UI for toggling that option?

Reply:
Is this solution fine? [`2de2f0f` (#966)](https://github.com/cloudfour/cloudfour.com-patterns/pull/966/commits/2de2f0f3d2ee6c030a71ffca27aa08bd8e9d5edd)

Reply:
@calebeby Yeah, that's way better!